apps/ex_nvr/lib/ex_nvr/events.ex

```diff
@@ -0,0 +1,53 @@
+defmodule ExNVR.Events do
+  alias Membrane.Event
+  alias ExNVR.Model.{Device, LPREvent}
+  alias ExNVR.Repo
+
+  def create_lpr_event(params, plate_picture \\ nil, full_picture \\ nil) do
```

Comment:
All events are dependent on the device. Update this function to accept the device as an argument.
```elixir
def create_lpr_event(device, params, ...) do
  # set the device id of params
  ...
end
```